README.md

```diff
@@ -1 +1,23 @@
-# balena-es-version
\ No newline at end of file
+# @balena/es-version
+
+This module allows specifying a target es version for other modules to require, currently supported target versions:
+* es3 (default)
+* es5
+* es2015
+* es2016
+* es2017
+* es2018
+* es2019
+* es2020
+* esnext
+
+To use it first require it before anything else and set the desired version:
+```js
+// Must be set before the first require of an @balena/es-version module
+require('@balena/es-version').set('es5')
+```
+and then the closest matching version will be used by any module that supports choosing the target version, with the priority being:
+1. exact match
+2. closest match under the desired version
+3. closest match over the desired version
```

Comment:
Maybe it's safer not to have this, and fail immediately. I can consciously set the version to something higher afterwards if I see an error.

Reply:
Is this essentially returning the lowest available build target of the library?

Eg: Pinejs-client-js-next only offers builds for es2015 & es2020 but the CLI sets the desired to be es5 (or doesn't set any preference, which makes es3 to be used as the desired version, since it's the default).
In which case we give back es2019 and the CLI takes its chances on whether it will work or not?

If so, then I guess that is makes sense, but we definitely need a comment so that we don't forget this case.

Reply:
For  
`get(['es2015', 'es1999', 'es2016']) / get(['es2015', 'es2050', 'es2016'])`  
the reasoning was the case of
`get(['es2015', 'es2018', 'es2021'])`
where the version might be valid in the future, but if they're using @balena/es-version to attempt to get a version that it doesn't know about then it should update its minimum @balena/es-version version

Reply:
I guess so that this should be the reason :)
But the consumer does have "ways" (arguably not so friendly) to bump & dedupe the @balena/es-version used by its dependencies, so imo this shouldn't be a blocker.